### PR TITLE
Syslog regex parser example documentation

### PR DESCRIPTION
**Pull request by lzammit**
# Description
Give two example for the syslog parser configuration

# Impacts
Doc

# Delete branch after merge
YES

### CODE REVIEW

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 1]
```diff
@@ -2775,7 +2775,160 @@ Action -
 
     modify_node:  mac, $mac, notes, $notes
 
+uricata example:
```
Comment (julsemaan) [inline docs/PacketFence_Administration_Guide.asciidoc]: suricata

Comment (extrafu) [inline docs/PacketFence_Administration_Guide.asciidoc]: uricata? Do you have an urinary infection?

Comment (julsemaan) [reply docs/PacketFence_Administration_Guide.asciidoc]: I think its for urinary violations

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 2]
```diff
@@ -2775,7 +2775,160 @@ Action -
...
+PacketFence already contains a syslog parser for Suricata. This is an example to raise a violation from a syslog alert on the Suricata SID.
+
+The first step is to create the syslog regex parser and then create the violation.
+
+Syslog regex parser configuration:
+----------------------------------
+
+To create the syslog regex parser you will need to go to *Configuration -> Compliance -> Syslog Parsers -> Add a Syslog Parser -> regex*
+
+Here the configuration of the syslog regex parser:
```
Comment (extrafu) [inline docs/PacketFence_Administration_Guide.asciidoc]: Here is the ...

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 3]
```diff
@@ -2775,7 +2775,160 @@ Action -
...
+
+    02/26/2017-14:29:00.524309  [**] [1:2000340:10] ET P2P Kaaza Media desktop p2pnetworking.exe Activity [**] [Classification: Potential Corporate Privacy Violation] [Priority: 1] {UDP} 173.194.7.75:443 -> 1.2.3.4:46742
+
+In order to have a correct match in the rule, you will need to have a valid iplog entry in the database. Put the string in the test box and then click on the 'RUN TEST' button, you should get:
+
+    Click to see actions for - 02/26/2017-14:29:00.524309 [**] [1:2000340:10] ET P2P Kaaza Media desktop p2pnetworking.exe Activity [**] [Classification: Potential Corporate Privacy Violation] [Priority: 1] {UDP} 173.194.7.75:443 -> 1.2.3.4:46742
+ 
+    - ET P2P Kaaza Media desktop p2pnetworking.exe : trigger_violation('mac', '00:11:22:33:44:55', 'tid', '2000340', 'type', 'detect')
+
+We can see that PacketFence will execute the violation on the mac address 00:11:22:33:44:55.
```
Comment (extrafu) [inline docs/PacketFence_Administration_Guide.asciidoc]: MAC address

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 4]
```diff
@@ -2775,7 +2775,160 @@ Action -
...
+    Grace: 5 minutes
+    Template: p2p.html
+
+Click on the SAVE button.
+
+Now you will need to restart the pfqueue and the pfdetect services.
+
+    /usr/local/pf/bin/pfcmd service pfqueue restart
+    
+    /usr/local/pf/bin/pfcmd service pfdetect restart (Make sure that you have your pipe file otherwise the process won't start)
```
Comment (extrafu) [inline docs/PacketFence_Administration_Guide.asciidoc]: Put the "Make sure ... " on a separate line, remove the parenthesis.

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 5]
```diff
@@ -2775,7 +2775,160 @@ Action -
...
+    Grace: 5 minutes
+    Template: generic.html
+
+Click on the SAVE button.
+
+Now you will need to restart the pfqueue and the pfdetect services.
+
+    /usr/local/pf/bin/pfcmd service pfqueue restart
+    
+    /usr/local/pf/bin/pfcmd service pfdetect restart (Make sure that you have your pipe file otherwise the process won't start)
```
Comment (extrafu) [inline docs/PacketFence_Administration_Guide.asciidoc]: Same here.

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 6]
```diff
@@ -2775,7 +2775,160 @@ Action -
...
+    Apr  6 10:16:18 ubuntu CDS[83162]:type=violation device=172.20.20.193 threat="ssh trafic detected" direction=inbound sourceip=172.145.25.10 sourceport=22 destip=172.145.25.10 destport=42290 app=ssh timestamp=2017-04-06_10-16-18.965687 sid=1234
+
+In order to have a correct match in the rule, you will need to have a valid iplog entry in the database. Put the string in the test box and then click on the 'RUN TEST' button, you should get:
+
+    Results
+
+    Click to see actions for - Apr 6 10:16:18 ubuntu CDS[83162]:type=violation device=172.20.20.193 threat="ssh trafic detected" direction=inbound sourceip=172.145.25.10 sourceport=22 destip=172.145.25.10 destport=42290 app=ssh timestamp=2017-04-06_10-16-18.965687 sid=1234
+    - violation : trigger_violation('mac', '00:11:22:33:44:55', 'tid', '1234', 'type', 'detect')
+
+We can see that PacketFence will execute the violation on the mac address 00:11:22:33:44:55.
```
Comment (extrafu) [inline docs/PacketFence_Administration_Guide.asciidoc]: MAC address

**Pull request merged**